inventory-generation/tower-jobs-init/inventory/hosts

```diff
@@ -0,0 +1,3 @@
+[tower-jobs-init]
```

Comment:
We should use `_` in group names as Ansible is showing a warning when using `-` (we have a job to convert everything old, but let's use `_` for newly introduced groups).

Reply:
Updated this one to `tower_jobs_init`